### PR TITLE
Refactor detector.py

### PR DESCRIPTION
**Pull request by Nathan24zz**
refactor detector.py so it's compatible with ninshiki_interfaces msg

### CODE REVIEW

File: ninshiki/detector.py [hunk 1]
```diff
@@ -52,21 +51,21 @@ def __init__(self, node_name: str, topic_name: str, model_path: str):
             "publish detected images on "
             + self.detected_object_publisher.topic_name)
 
-    def listener_callback(self, message: MsgType):
+    def listener_callback(self, message):
+        self.rows = message.rows
+        self.cols = message.cols
```
Comment (threeal) [inline ninshiki/detector.py]: I think it's no longer needed.

**Review by threeal — DISMISSED** [review]
All ok, but with a comment that worth to be check.

**Review by threeal — APPROVED** [review]
Looks good to me.

**Review by maroqijalil — APPROVED** [review]
LGTM

**Pull request merged**